packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -185,7 +187,8 @@ export class TeamsUserCredential implements TokenCredential {
     options?: GetTokenOptions
   ): Promise<AccessToken | null> {
     validateScopesType(scopes);
-    const ssoToken = await this.getSSOToken();
+    const resources = (options as GetTeamsUserTokenOptions)?.resources;
```

Comment:
I guess the signature of `getToken` is unchanged in order to align with `TokenCredential` interface.
Do we need to add comment and example for this API feature? It looks like developers would need to pass the `resources` value in `options` parameter if they want to use full trust Teams apps.

Reply:
@tecton that's a great call, I've updated the comments to include more information about this for full trust apps along with an example of how to use this. Is there anything else I need update for the "codecov/project" task to run?

Reply:
Maybe you can merge 'dev' branch again to trigger the task.